py/README.rst

```diff
@@ -103,10 +103,8 @@ Run IPython with PySparkling
 Use PySparkling as Spark Package
 --------------------------------
 .. code-block:: bash
-
-	$SPARK_HOME/bin/spark-submit
-	--packages ai.h2o:sparkling-water-core_2.10:1.6.1
-	--py-files $SPARKLING_HOME/py/dist/pySparkling-1.6.1-py2.7.egg  ./py/examples/scripts/ChicagoCrimeDemo.py
+    export SPARKLING_EGG=$(ls $SPARKLING_HOME/py/dist/)
```

Comment:
True; let me work on it (later)

Thanks,
-Vlad

On Wed, Aug 31, 2016 at 2:10 PM, Michal Malohlava <notifications@github.com>
wrote:

> In py/README.rst
> <https://github.com/h2oai/sparkling-water/pull/66#discussion_r77074251>:
>
> > @@ -103,10 +103,8 @@ Run IPython with PySparkling
> >  Use PySparkling as Spark Package
> >  --------------------------------
> >  .. code-block:: bash
> > -
> > -	$SPARK_HOME/bin/spark-submit
> > -	--packages ai.h2o:sparkling-water-core_2.10:1.6.1
> > -	--py-files $SPARKLING_HOME/py/dist/pySparkling-1.6.1-py2.7.egg  ./py/examples/scripts/ChicagoCrimeDemo.py
> > +    export SPARKLING_EGG=$(ls $SPARKLING_HOME/py/dist/)
>
> ls $SPARKLING_HOME/py/dist/ or ls $SPARKLING_HOME/py/dist/*.egg - what if
> the directory contains some garbage files
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/66/files/e2ac36c0116dc048f50809524a2e2c26617255fc#r77074251>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuK3bYB8NveBVxhKv5Ugb3pCecEo7zks5qle3ggaJpZM4Jw-AK>
> .
>